[2023-01-12T11:41:25.487179+00:00] deprecation.INFO: User Deprecated: The "Monolog\Logger" class is considered final. It may change without further notice as of its next major version. You should not extend it from "Symfony\Bridge\Monolog\Logger". {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Monolog\\Logger\" class is considered final. It may change without further notice as of its next major version. You should not extend it from \"Symfony\\Bridge\\Monolog\\Logger\". at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/error-handler/DebugClassLoader.php:331)"} []
[2023-01-12T11:48:51.365446+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:185)"} []
[2023-01-12T11:48:51.442446+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"6fa835"},"request_uri":"http://localhost:8000/_wdt/6fa835","method":"GET"} []
[2023-01-12T11:48:54.743884+00:00] request.INFO: Matched route "app_login". {"route":"app_login","route_parameters":{"_route":"app_login","_controller":"App\\Controller\\LoginController::index"},"request_uri":"http://localhost:8000/login","method":"GET"} []
[2023-01-12T11:48:54.755672+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T11:48:54.755754+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T11:48:54.755795+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T11:48:54.809176+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/test.css" (from "http://localhost:8000/login")" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/test.css\" (from \"http://localhost:8000/login\") at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/test.css/\". at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2023-01-12T11:48:54.867384+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9dcfd5"},"request_uri":"http://localhost:8000/_wdt/9dcfd5","method":"GET"} []
[2023-01-12T11:51:00.236398+00:00] request.INFO: Matched route "app_register". {"route":"app_register","route_parameters":{"_route":"app_register","_controller":"App\\Controller\\RegistrationController::register"},"request_uri":"http://localhost:8000/register","method":"GET"} []
[2023-01-12T11:51:00.238323+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T11:51:00.238456+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T11:51:00.238514+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T11:51:00.354878+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/test.css" (from "http://localhost:8000/register")" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/test.css\" (from \"http://localhost:8000/register\") at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/test.css/\". at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2023-01-12T11:51:00.374641+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"a49648"},"request_uri":"http://localhost:8000/_wdt/a49648","method":"GET"} []
[2023-01-12T16:22:52.043429+00:00] request.INFO: Matched route "app_register". {"route":"app_register","route_parameters":{"_route":"app_register","_controller":"App\\Controller\\RegistrationController::register"},"request_uri":"http://localhost:8000/register","method":"GET"} []
[2023-01-12T16:22:52.047375+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T16:22:52.047432+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:22:52.047468+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:22:52.108464+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/test.css" (from "http://localhost:8000/register")" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/test.css\" (from \"http://localhost:8000/register\") at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/test.css/\". at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2023-01-12T16:22:52.138795+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"95544f"},"request_uri":"http://localhost:8000/_wdt/95544f","method":"GET"} []
[2023-01-12T16:22:53.375834+00:00] request.INFO: Matched route "app_register". {"route":"app_register","route_parameters":{"_route":"app_register","_controller":"App\\Controller\\RegistrationController::register"},"request_uri":"http://localhost:8000/register","method":"GET"} []
[2023-01-12T16:22:53.376651+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T16:22:53.376747+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:22:53.376797+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:22:53.417397+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/test.css" (from "http://localhost:8000/register")" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/test.css\" (from \"http://localhost:8000/register\") at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/test.css/\". at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2023-01-12T16:22:53.442298+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"271dc3"},"request_uri":"http://localhost:8000/_wdt/271dc3","method":"GET"} []
[2023-01-12T16:22:53.588113+00:00] request.INFO: Matched route "app_register". {"route":"app_register","route_parameters":{"_route":"app_register","_controller":"App\\Controller\\RegistrationController::register"},"request_uri":"http://localhost:8000/register","method":"GET"} []
[2023-01-12T16:22:53.589450+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T16:22:53.589658+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:22:53.589714+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:22:53.635820+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/test.css" (from "http://localhost:8000/register")" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/test.css\" (from \"http://localhost:8000/register\") at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/test.css/\". at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2023-01-12T16:22:53.660500+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"82667b"},"request_uri":"http://localhost:8000/_wdt/82667b","method":"GET"} []
[2023-01-12T16:22:56.905852+00:00] request.INFO: Matched route "lucky". {"route":"lucky","route_parameters":{"_route":"lucky","_controller":"App\\Controller\\HomeController::lucky"},"request_uri":"http://localhost:8000/lucky","method":"GET"} []
[2023-01-12T16:22:56.908119+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T16:22:56.908268+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:22:56.908326+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:22:56.918122+00:00] security.DEBUG: Access denied, the user is not fully authenticated; redirecting to authentication entry point. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException(code: 403): Access Denied. at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/security-http/Firewall/AccessListener.php:97)"} []
[2023-01-12T16:22:56.919036+00:00] security.DEBUG: Calling Authentication entry point. {"entry_point":{"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator":[]}} []
[2023-01-12T16:22:56.930899+00:00] request.INFO: Matched route "app_login". {"route":"app_login","route_parameters":{"_route":"app_login","_controller":"App\\Controller\\LoginController::index"},"request_uri":"http://localhost:8000/login","method":"GET"} []
[2023-01-12T16:22:56.931302+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T16:22:56.931345+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:22:56.931371+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:22:56.961385+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/test.css" (from "http://localhost:8000/login")" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/test.css\" (from \"http://localhost:8000/login\") at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/test.css/\". at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2023-01-12T16:22:56.985468+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"63de55"},"request_uri":"http://localhost:8000/_wdt/63de55","method":"GET"} []
[2023-01-12T16:23:07.465857+00:00] request.INFO: Matched route "app_login". {"route":"app_login","route_parameters":{"_route":"app_login","_controller":"App\\Controller\\LoginController::index"},"request_uri":"http://localhost:8000/login","method":"POST"} []
[2023-01-12T16:23:07.467899+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T16:23:07.468064+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:23:07.499218+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"app","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"14","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"app","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"14","charset":"utf8"}} []
[2023-01-12T16:23:07.506916+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[08006] [7] connection to server at "127.0.0.1", port 5432 failed: Connection refused 	Is the server running on that host and accepting TCP/IP connections?" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php line 77 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 7): An exception occurred in the driver: SQLSTATE[08006] [7] connection to server at \"127.0.0.1\", port 5432 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections? at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:77)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 7): SQLSTATE[08006] [7] connection to server at \"127.0.0.1\", port 5432 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections? at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 7): SQLSTATE[08006] [7] connection to server at \"127.0.0.1\", port 5432 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections? at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:29)"} []
[2023-01-12T16:23:07.625275+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"1e1df3"},"request_uri":"http://localhost:8000/_wdt/1e1df3","method":"GET"} []
[2023-01-12T16:23:23.369003+00:00] request.INFO: Matched route "app_login". {"route":"app_login","route_parameters":{"_route":"app_login","_controller":"App\\Controller\\LoginController::index"},"request_uri":"http://localhost:8000/login","method":"GET"} []
[2023-01-12T16:23:23.370631+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T16:23:23.370716+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:23:23.370773+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:23:23.404345+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/test.css" (from "http://localhost:8000/login")" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/test.css\" (from \"http://localhost:8000/login\") at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/test.css/\". at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2023-01-12T16:23:23.431736+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"3b7191"},"request_uri":"http://localhost:8000/_wdt/3b7191","method":"GET"} []
[2023-01-12T16:23:25.847605+00:00] request.INFO: Matched route "app_login". {"route":"app_login","route_parameters":{"_route":"app_login","_controller":"App\\Controller\\LoginController::index"},"request_uri":"http://localhost:8000/login","method":"POST"} []
[2023-01-12T16:23:25.849427+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T16:23:25.849543+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:23:25.855461+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"app","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"14","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"app","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"14","charset":"utf8"}} []
[2023-01-12T16:23:25.859221+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[08006] [7] connection to server at "127.0.0.1", port 5432 failed: Connection refused 	Is the server running on that host and accepting TCP/IP connections?" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php line 77 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 7): An exception occurred in the driver: SQLSTATE[08006] [7] connection to server at \"127.0.0.1\", port 5432 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections? at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:77)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 7): SQLSTATE[08006] [7] connection to server at \"127.0.0.1\", port 5432 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections? at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 7): SQLSTATE[08006] [7] connection to server at \"127.0.0.1\", port 5432 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections? at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:29)"} []
[2023-01-12T16:23:25.973052+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"76d92b"},"request_uri":"http://localhost:8000/_wdt/76d92b","method":"GET"} []
[2023-01-12T16:23:42.011179+00:00] request.INFO: Matched route "app_login". {"route":"app_login","route_parameters":{"_route":"app_login","_controller":"App\\Controller\\LoginController::index"},"request_uri":"http://127.0.0.1:8000/login","method":"GET"} []
[2023-01-12T16:23:42.012496+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T16:23:42.012723+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:23:42.012779+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:23:42.047900+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8000/test.css" (from "http://127.0.0.1:8000/login")" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8000/test.css\" (from \"http://127.0.0.1:8000/login\") at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/test.css/\". at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2023-01-12T16:23:42.076059+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f9774f"},"request_uri":"http://127.0.0.1:8000/_wdt/f9774f","method":"GET"} []
[2023-01-12T16:23:45.763785+00:00] request.INFO: Matched route "app_login". {"route":"app_login","route_parameters":{"_route":"app_login","_controller":"App\\Controller\\LoginController::index"},"request_uri":"http://127.0.0.1:8000/login","method":"POST"} []
[2023-01-12T16:23:45.765926+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T16:23:45.766027+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:23:45.770178+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"app","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"14","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"app","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"14","charset":"utf8"}} []
[2023-01-12T16:23:45.772864+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[08006] [7] connection to server at "127.0.0.1", port 5432 failed: Connection refused 	Is the server running on that host and accepting TCP/IP connections?" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php line 77 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 7): An exception occurred in the driver: SQLSTATE[08006] [7] connection to server at \"127.0.0.1\", port 5432 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections? at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:77)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 7): SQLSTATE[08006] [7] connection to server at \"127.0.0.1\", port 5432 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections? at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 7): SQLSTATE[08006] [7] connection to server at \"127.0.0.1\", port 5432 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections? at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:29)"} []
[2023-01-12T16:23:45.901043+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"1dd2b9"},"request_uri":"http://127.0.0.1:8000/_wdt/1dd2b9","method":"GET"} []
[2023-01-12T16:27:47.629297+00:00] request.INFO: Matched route "app_login". {"route":"app_login","route_parameters":{"_route":"app_login","_controller":"App\\Controller\\LoginController::index"},"request_uri":"http://127.0.0.1:8000/login","method":"POST"} []
[2023-01-12T16:27:47.630788+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T16:27:47.630862+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T16:27:47.634621+00:00] doctrine.INFO: Connecting with parameters array{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"app","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"14","charset":"utf8"} {"params":{"url":"<redacted>","driver":"pdo_pgsql","host":"127.0.0.1","port":5432,"user":"app","password":"<redacted>","driverOptions":[],"defaultTableOptions":[],"dbname":"app","serverVersion":"14","charset":"utf8"}} []
[2023-01-12T16:27:47.637652+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\DBAL\Exception\ConnectionException: "An exception occurred in the driver: SQLSTATE[08006] [7] connection to server at "127.0.0.1", port 5432 failed: Connection refused 	Is the server running on that host and accepting TCP/IP connections?" at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php line 77 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 7): An exception occurred in the driver: SQLSTATE[08006] [7] connection to server at \"127.0.0.1\", port 5432 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections? at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:77)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 7): SQLSTATE[08006] [7] connection to server at \"127.0.0.1\", port 5432 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections? at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 7): SQLSTATE[08006] [7] connection to server at \"127.0.0.1\", port 5432 failed: Connection refused\n\tIs the server running on that host and accepting TCP/IP connections? at /Users/coulbeaf/PhpstormProjects/sae3_garage_symfony/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:29)"} []
[2023-01-12T16:27:47.764053+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"68f560"},"request_uri":"http://127.0.0.1:8000/_wdt/68f560","method":"GET"} []
[2023-01-12T16:48:48.166627+00:00] php.DEBUG: Warning: file(/proc/mounts): Failed to open stream: No such file or directory {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/Users/coulbeaf/PhpstormProjects/sae4/vendor/symfony/framework-bundle/Command/CacheClearCommand.php","line":209,"trace":[{"file":"/Users/coulbeaf/PhpstormProjects/sae4/vendor/symfony/framework-bundle/Command/CacheClearCommand.php","line":162,"function":"isNfs","class":"Symfony\\Bundle\\FrameworkBundle\\Command\\CacheClearCommand","type":"->"}],"count":1}}} []
[2023-01-12T17:35:48.825451+00:00] php.DEBUG: Warning: filemtime(): stat failed for /Users/coulbeaf/PhpstormProjects/sae4/config/routes/framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/config/Resource/FileResource.php","line":61,"trace":[{"file":"/Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":44,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2023-01-12T17:35:48.913652+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8001/"" at /Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8001/\" at /Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:185)"} []
[2023-01-12T17:35:49.023118+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"76d8ca"},"request_uri":"http://127.0.0.1:8001/_wdt/76d8ca","method":"GET"} []
[2023-01-12T17:35:49.050178+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8001/favicon.ico" (from "http://127.0.0.1:8001/")" at /Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8001/favicon.ico\" (from \"http://127.0.0.1:8001/\") at /Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2023-01-12T17:35:53.727402+00:00] request.INFO: Matched route "app_register". {"route":"app_register","route_parameters":{"_route":"app_register","_controller":"App\\Controller\\RegistrationController::register"},"request_uri":"http://127.0.0.1:8001/register","method":"GET"} []
[2023-01-12T17:35:53.733835+00:00] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":1} []
[2023-01-12T17:35:53.733916+00:00] security.DEBUG: Checking support on authenticator. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T17:35:53.733953+00:00] security.DEBUG: Authenticator does not support the request. {"firewall_name":"main","authenticator":"Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"} []
[2023-01-12T17:35:53.835172+00:00] php.DEBUG: Warning: filemtime(): stat failed for /Users/coulbeaf/PhpstormProjects/sae4/vendor/symfony/validator/Resources/translations/validators.en.xlf {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/config/Resource/FileResource.php","line":61,"trace":[{"file":"/Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":44,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2023-01-12T17:35:53.882953+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1:8001/test.css" (from "http://127.0.0.1:8001/register")" at /Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://127.0.0.1:8001/test.css\" (from \"http://127.0.0.1:8001/register\") at /Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/http-kernel/EventListener/RouterListener.php:128)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/test.css/\". at /Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2023-01-12T17:35:53.927044+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"6dfc9d"},"request_uri":"http://127.0.0.1:8001/_wdt/6dfc9d","method":"GET"} []
[2023-01-12T17:56:33.349932+00:00] deprecation.INFO: User Deprecated: The "Monolog\Logger" class is considered final. It may change without further notice as of its next major version. You should not extend it from "Symfony\Bridge\Monolog\Logger". {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Monolog\\Logger\" class is considered final. It may change without further notice as of its next major version. You should not extend it from \"Symfony\\Bridge\\Monolog\\Logger\". at /Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/error-handler/DebugClassLoader.php:331)"} []
[2023-01-12T17:58:31.956337+00:00] deprecation.INFO: User Deprecated: The "Monolog\Logger" class is considered final. It may change without further notice as of its next major version. You should not extend it from "Symfony\Bridge\Monolog\Logger". {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Monolog\\Logger\" class is considered final. It may change without further notice as of its next major version. You should not extend it from \"Symfony\\Bridge\\Monolog\\Logger\". at /Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/error-handler/DebugClassLoader.php:331)"} []
[2023-01-12T18:15:16.900771+00:00] deprecation.INFO: User Deprecated: The "Monolog\Logger" class is considered final. It may change without further notice as of its next major version. You should not extend it from "Symfony\Bridge\Monolog\Logger". {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Monolog\\Logger\" class is considered final. It may change without further notice as of its next major version. You should not extend it from \"Symfony\\Bridge\\Monolog\\Logger\". at /Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/error-handler/DebugClassLoader.php:331)"} []
[2023-01-12T18:15:23.357285+00:00] console.DEBUG: Command "make:crud Personne" exited with code "1" {"command":"make:crud Personne","code":1} []
[2023-01-12T18:15:41.649009+00:00] console.DEBUG: Command "make:crud Match" exited with code "1" {"command":"make:crud Match","code":1} []
[2023-01-12T18:15:49.146314+00:00] console.DEBUG: Command "make:crud Match" exited with code "1" {"command":"make:crud Match","code":1} []
[2023-01-12T18:15:53.030108+00:00] console.DEBUG: Command "make:crud Matchs" exited with code "1" {"command":"make:crud Matchs","code":1} []
[2023-01-12T18:16:11.383317+00:00] deprecation.INFO: User Deprecated: The "Monolog\Logger" class is considered final. It may change without further notice as of its next major version. You should not extend it from "Symfony\Bridge\Monolog\Logger". {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Monolog\\Logger\" class is considered final. It may change without further notice as of its next major version. You should not extend it from \"Symfony\\Bridge\\Monolog\\Logger\". at /Users/coulbeaf/PhpstormProjects/Sae-S4.A.01/SaeA01/Framework/Symfony/vendor/symfony/error-handler/DebugClassLoader.php:331)"} []
